src/components/ReviewSwap.tsx

```diff
@@ -103,21 +114,21 @@ function ReviewSwap({ onClose, onConfirm, data }: Props): ReactElement {
                 />
               </svg>
             </button>
-            <span className="value floatRight">
+            <Typography component="span" ml="auto" mr={0}>
               {formatBNToString(data.exchangeRateInfo.exchangeRate, 18, 6)}
-            </span>
-          </div>
-          <div className="row">
-            <span className="title">{t("gas")}</span>
-            <span className="value floatRight">
+            </Typography>
+          </Box>
+          <Box display="flex">
+            <Typography component="span">{t("gas")}</Typography>
```

Comment:
We should be hiding gas - we don't currently have a good way of estimating this value and instead leave it up to the user's wallet